src/App.css

```diff
@@ -1,38 +1,203 @@
-body {
-  background-color: rgb(176, 202, 202);
+.nav-bar {
+  background-color: #fbfcfd;
+  padding: 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav-bar ul {
+  list-style: none;
+  display: flex;
+  align-items: center;
+}
+
+.nav-bar ul h2 {
+  color: #0290ff;
+}
+
+.nav-bar ul li {
+  margin-right: 1rem;
 }
 
-h1,
-h2,
-h3,
-p,
-ul,
-li {
+.nav-bar ul li:last-child {
+  margin-right: 0;
+}
+
+.nav-bar a {
+  color: rgb(8, 8, 8);
+  text-decoration: none;
+  font-size: 1.2rem;
+}
+
+.avatar {
+  margin-right: 1rem;
+}
+
+hr {
   margin: 0;
-  padding: 0;
+  padding-bottom: 20px;
+  border: 0;
+  border-top: 1px solid #ccc;
+}
+
+.bar-list {
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
 }
 
-.navbar {
-  background-color: #333;
-  color: white;
+.book-details {
   display: flex;
-  justify-content: center;
+  justify-content: space-between;
   align-items: center;
-  padding: 10px;
+  /* padding: 1rem; */
+  padding-top: 20px;
+  margin-left: 10%;
+  border-radius: 4px;
+  border: solid 1px var(--white-three);
+  background-color: #fdfbfb;
+  width: 80%;
+}
+
+.book-details:hover {
+  border: solid 1px var(--white-three);
+  background-color: #f5f5f5;
 }
 
-.nav-links {
+.book-details li {
   list-style: none;
+}
+
+.Title1 {
+  width: 12.313rem;
+  height: 1.813rem;
+  /* margin: 0.188rem 20.188rem 0 0; */
+  margin-left: 16px;
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure that your design layout, colors, padding, and margin of elements (cards, buttons, form, and navigation) match the provided design on [Zeplin](https://zeplin.io/) :pray: 
  ### Your design
  ![image](https://github.com/Eccb7/book-store/assets/92303891/e5b0f9b9-1ac4-4dcc-b2cd-5c90f6062173)
  
    ### Provided Design
  ![image](https://github.com/Eccb7/book-store/assets/92303891/47d5e202-6cdc-4ddc-a896-0badca37fee7)